timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -36,12 +36,18 @@ types:
   definitions:
     default-package: com.palantir.atlasdb.timelock.api
     objects:
+      ConjureIdentifiedVersion:
```

Comment:
nit: the Lock Watch stuff never really had a non-conjurized (?) form, so it's probably fine to just name this `IdentifiedVersion`

Reply:
So problem is that that IdentifiedVersion is already defined in the lock-api-objects. So until we clean that up, I think it makes sense to keep this as ConjureIdentifiedVersion.